makefiles/Makefile.dotnet.mk

```diff
@@ -446,7 +446,7 @@ build: $(SOURCE) $(SOURCE)proj $(DOTNET_ORTOOLS_NUPKG)
 
 .PHONY: run # Run a .Net C# program (using 'dotnet run').
 run: build
-	"$(DOTNET_BIN)" run $(DOTNET_BUILD_ARGS) --no-build $(ARGS) --project $(SOURCE_PATH)proj
```

Comment:
this is the **run** target which depends on the **build** target -> no need to suppress the option here IMHO

Reply:
`run` isn't building by default, I don't know wheather it worked before but with `no-build` the samples test fail now because of the missing binary.

Reply:
You are right, I will check that after holiday.

Reply:
Ok, I readded `no-build` option to **run** target and it works.